app/src/main/res/values/strings.xml

```diff
@@ -319,6 +319,14 @@
   <!-- Appointment reminder -->
   <string name="appointmentreminder_call_later_in">Call later in</string>
   <string name="appointmentreminder_done">Done</string>
+  <plurals name="appointmentreminder_days">
+    <item quantity="one">%d day</item>
+    <item quantity="other">%d days</item>
+  </plurals>
+  <plurals name="appointmentreminder_weeks">
+    <item quantity="one">%d week</item>
+    <item quantity="other">%d weeks</item>
+  </plurals>
```

Comment:
We're going to be forcing arabic numerals in the UI going forward. This means that the placeholders should be using string placeholders instead of integer placeholders. Please update this?

See: https://docs.google.com/document/d/1v5siRSZD3YgnWpPJ9dgEvt7Z7x_O-9mc0mh78y0kUD0/edit#

Reply:
Done. Forgot about this change.